modules/gateways/callback/blockonomics.php

```diff
@@ -78,9 +77,15 @@
 $orderNote = "";
 
 if($value < $bits) {
-	$orderNote .= "Warning: Invoice cancelled as Paid Amount was less than expected\r";
-	$blockonomics->updateInvoiceStatus($invoiceId, "Cancelled");
-	$blockonomics->updateOrderStatus($true_order_id, 'Cancelled');
+	$invoiceNote .= "Warning: Paid Amount was less than the expected amount. Invoice will not be marked paid before the whole amount is added.\r";
```

Comment:
This note is uncessary and can be removed. WHMCS is clearly showing the balance amount